docs/source/getting_started/using_entitysets.ipynb

```diff
@@ -71,7 +71,19 @@
     "\n",
     "* The ``index`` parameter specifies the column that uniquely identifies rows in the dataframe.\n",
     "* The ``time_index`` parameter tells Featuretools when the data was created.\n",
-    "* The ``logical_types`` parameter indicates that \"product_id\" should be interpreted as a Categorical column, even though it is just an integer in the underlying data."
+    "* The ``logical_types`` parameter indicates that \"product_id\" should be interpreted as a Categorical column, even though it is just an integer in the underlying data.\n",
+    "\n",
+    "You can also use a setter on the ``EntitySet`` object to add dataframes\n",
```

Comment:
I think it might flow better if we move this a little lower in the section, after we demonstrate how to use `add_dataframe`.

Also, what do you think about putting this in a Note callout, like we do below with the description of `plot`?

Reply:
I tried to put this into a note callout, but they don't support code blocks or lists. I guess I could reword it

Reply:
Ah, ok, didn't realize that. No big deal on the note formatting, but if we can try somehow separate it from the flow of the example, that might be good.

Reply:
Actually lists work, but can't get codeblocks to work. I reformatted it